### DISCUSSION TITLE
Dialogic displays uncalled for dialog box at start of game

### DISCUSSION THREAD

**Discussion by MyChemicalAddiction** [dialogic-godot/dialogic#972] [Q&A] [answered]
I am using the latest version. 
I implemented dialogic in my game. 
When I run the game, I get the following dialog box at the bottom of my screen:

![image](https://user-images.githubusercontent.com/60291234/177002984-23d06a3d-ae45-400d-87fd-341312f3106c.png)

I have confirmed it is not because of any of my code.
Upon investigation I have found that commenting out these specific lines in DialogProxy.gd, at res://addons/dialogic/Nodes/DialogProxy.gd fixes this problem:

![image](https://user-images.githubusercontent.com/60291234/177002961-9f876184-540b-490c-9ab1-ad656cd6e9ca.png)

I would like someone to help me understand the purpose of these lines of code, whether they are the root of my issue and what problems could arise from commenting them out.

**Comment by zaknafean** [comment, marked as answer]
You are likely using the DialogNode in your scene. The node always plays immediately, as its main purpose is to test things rapidly. 

Generally you should be instantiating Dialog with Dialogic.start(<timeline name here>) from the appropriate spot of your code.

More or less using the DialogNode in your scene calls the DialogProxy, and it just does some stuff for you. You can comment them out if you like, especially if you never intend to use the DialogNode, but it'll be easier to just find the DialogNode and remove it from the scene. You really don't need it.

**Reply by MyChemicalAddiction** [reply]
perfect answer, and it indeed was in my scene. thanks for your help
